Sources/SecureXPC/Server/XPCMachServer.swift

```diff
@@ -14,21 +14,31 @@ internal class XPCMachServer: XPCServer, NonBlockingStartable {
     
     /// Name of the service.
     private let machServiceName: String
-    /// Receives new incoming connections, created once the server is started.
-    private var listenerConnection: xpc_connection_t?
+    /// Receives new incoming connections
+    private var listenerConnection: xpc_connection_t
     /// Determines if an incoming request will be accepted based on the provided client requirements
     private let messageAcceptor: SecureMessageAcceptor
 
     /// This should only ever be called from `getXPCMachServer(...)` so that client requirement invariants are upheld.
     private init(machServiceName: String, clientRequirements: [SecRequirement]) {
         self.machServiceName = machServiceName
         self.messageAcceptor = SecureMessageAcceptor(requirements: clientRequirements)
+        
+        // Attempts to bind to the Mach service. If this isn't actually a Mach service a EXC_BAD_INSTRUCTION will occur.
```

Comment:
This is where the separation of concerns among SecureXPC/EmbeddedPropertyList/Blessed can be a little annoying.

Ideally, I'd love to use the facilities of `Blessed` to verify the caller is a mock service, and throw a prettier error.

Reply:
Actually `Blessed` isn't capable of knowing if the caller is a Mach service. I believe the functionality to determine that was exposed by the [`SMJobCopyDictionary(_:_:)` function](https://developer.apple.com/documentation/servicemanagement/1431092-smjobcopydictionary) which Apple deprecating many years ago. I don't understand why, but Apple deprecated most of the the Service Management framework and never introduced any replacements.